app/gui2/src/assets/base.css

```diff
@@ -57,6 +57,11 @@
   --right-dock-default-width: 40%;
 }
 
+/* Constants */
+:root {
```

Comment:
Not sure we need the `--pi` variable here: 
https://developer.mozilla.org/en-US/docs/Web/CSS/calc-constant#pi